src/applications/vaos/components/AddToCalendar.jsx

```diff
@@ -103,6 +108,95 @@ function generateICS(
   ].join('\r\n');
 }
 
+/**
+ * @summary Function that returns a collection of ICS key/value pairs.
+ * @description Only a subset of the ICS object specification is used.
+ * The following properties/keys are allowed:
+ *
+ * <ul>
+ * <li>BEGIN</li>
+ * <li>VERSION</li>
+ * <li>PRODID</li>
+ * <li>BEGIN</li>
+ * <li>UID</li>
+ * <li>SUMMARY</li>
+ * <li>DESCRIPTION</li>
+ * <li>LOCATION</li>
+ * <li>DTSTAMP</li>
+ * <li>DTSTART</li>
+ * <li>DTEND</li>
+ * <li>END</li>
+ * <li>END</li>
+ * </ul>
+ *
+ * Values are retreived from the returned collection using the defined keys. Values returned from
+ * the collection can be of type string or array for keys with multiply values. The following keys have
+ * multiple values:
+ *
+ * <ul>
+ * <li>BEGIN</li>
+ * <li>END</li>
+ * </ul>
+ *
+ * The following key:
+ * <ul>
+ * <li>FORMATTED_TEXT</li>
+ * </ul>
+ *
+ * is a special key used to access the formatted description text.
+ *
+ * @export
+ * @param {string} buffer - ICS calendar string
+ * @returns Returns a collection of ICS key/value pairs.
+ */
+export function getICSTokens(buffer) {
```

Comment:
I think we should move this function to another file. Since it's in with the AddToCalendar component, it may get included in the bundle we ship to users, even though it's only used in tests. Either somewhere in the tests folder or a new file in the utils folder would work, I think.